added 5 new bonfires

storbeck opened the pull request:
Created 5 new bonfires, four level 2 and one level 4

storbeck committed:
added 5 new bonfires

storbeck committed:
Updated language and added more test cases

terakilobyte commented:
This looks great so far @storbeck , thanks for the contribution. If I could ask one more thing, please include links to relevant MDN resources that @aldraco provided so we have a unified approach.

storbeck committed:
Updated from upstream

storbeck committed:
Forgot comma on the end of the new entry to bonfireMDNlinks.js

storbeck commented:
@terakilobyte Thanks! Looks like this was behind a few commits. I've updated it to be current, added in some MDN links and also created a new MDN link for `parseInt()`
